scri/SpEC/memory.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
```

Comment:
So far this script just defines functions, so this line isn't needed at the current time.